pennylane/devices/default_clifford.py

```diff
@@ -103,28 +104,49 @@
     "StatePrep": None,
     "Snapshot": None,
     "Barrier": None,
+    "PauliError": "CORRELATED_ERROR",
+    "BitFlip": "X_ERROR",
+    "PhaseFlip": "Z_ERROR",
+    "DepolarizingChannel": "DEPOLARIZE1",
 }
 
 
 def operation_stopping_condition(op: qml.operation.Operator) -> bool:
     """Specifies whether an operation is accepted by ``DefaultClifford``."""
-    return op.name in _GATE_OPERATIONS
+    return op.name in _OPERATIONS_MAP
 
 
 def observable_stopping_condition(obs: qml.operation.Operator) -> bool:
     """Specifies whether an observable is accepted by ``DefaultClifford``."""
-    return obs.name in _MEAS_OBSERVABLES
+    return obs.name in _OBSERVABLES_MAP
+
+
+@qml.transform
+def _validate_channels(tape, name="device"):
+    """Validates the channels for a circuit."""
+    if not tape.shots and any(isinstance(op, qml.operation.Channel) for op in tape.operations):
+        raise qml.DeviceError(f"Channel not supported on {name} without finite shots.")
```

Comment:
[non-blocking] This could be two separate errors.

Reply:
These two conditions are kept combined because we cannot support `Channels` only if the first condition is met i.e. analytical case. One can use channels if they are in the `finite-shot` region. On the other hand, if an unsupported channel is given, it is already being checked before the following check is performed in the `preprocess` method.